src/lib/components/workflow/dropdown-filter/workflow-datetime-filter.svelte

```diff
@@ -0,0 +1,233 @@
+<script lang="ts">
+  import { timeFormat } from '$lib/stores/time-format';
+  import { capitalize } from '$lib/utilities/format-camel-case';
+  import {
+    addHours,
+    addMinutes,
+    addSeconds,
+    endOfDay,
+    formatISO,
+    startOfDay,
+  } from 'date-fns';
+
+  import { columnOrderedDurations } from '$lib/utilities/to-duration';
+
+  import Select from '$lib/holocene/select/select.svelte';
+  import Option from '$lib/holocene/select/option.svelte';
+  import CustomSplitButton from '$lib/holocene/custom-split-button.svelte';
+  import { workflowFilters, workflowSorts } from '$lib/stores/filters';
+  import DatePicker from '$lib/holocene/date-picker.svelte';
+  import Button from '$lib/holocene/button.svelte';
+  import { onMount } from 'svelte';
+  import TimePicker from '$lib/holocene/time-picker.svelte';
+  import { updateQueryParamsFromFilter } from '$lib/utilities/query/to-list-workflow-filters';
+  import { page } from '$app/stores';
+  import Icon from '$lib/holocene/icon/icon.svelte';
+
+  let custom = false;
+  let value = 'All Time';
+  let timeField = 'StartTime';
+
+  onMount(() => {
```

Comment:
Filter isn't being reset after search input is cleared.

![Screen Shot 2022-10-12 at 10 39 01 AM](https://user-images.githubusercontent.com/15069288/195400391-eab305b0-c4b0-4cb7-9c05-4b22cbda14e9.png)